### PR TITLE
[DATALAD RUNCMD] BF: remove spurious . from url -- could not find in compiled materials

### PR DESCRIPTION
**Pull request by yarikoptic**
What should ignorant in vue do to make sure this takes an effect?

### CODE REVIEW

File: ui/src/components/Home.vue [hunk 1]
```diff
@@ -19,7 +19,7 @@
             For every poster there is a dedicated Jitsi room, which would open in a "dedicated" new window/tab.
             Please search by poster number, keywords, names, etc.
             Posters with non-0 people on the video chat are shown with a persistent "online" counts. The counts are approximate and count only people attending through this page.
-            More info, sources, issues, PRs: <a href="https://github.com/datalad-datasets/ohbm2020-posters.">https://github.com/datalad-datasets/ohbm2020-posters.</a>
+            More info, sources, issues, PRs: <a href="https://github.com/datalad-datasets/ohbm2020-posters.">https://github.com/datalad-datasets/ohbm2020-posters</a>.
```
Comment (yarikoptic) [inline ui/src/components/Home.vue]: oh -- missed actual fix... will redo and force push

**Comment by yarikoptic** [comment]
@soichih @emdupre -- please take over to "compile"

**Comment by soichih** [comment]
OK. I will also add a link to "old" version while at it

**Comment by soichih** [comment]
PR #117 takes care of this.

**Pull request closed without merging**